src/Microsoft.AspNetCore.WebSockets/Microsoft.AspNetCore.WebSockets.csproj

```diff
@@ -14,6 +14,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="$(AspNetCoreVersion)" />
     <PackageReference Include="Microsoft.Extensions.Options" Version="$(AspNetCoreVersion)" />
+    <PackageReference Include="System.Numerics.Vectors" Version="$(CoreFxVersion)" />
```

Comment:
We didn't used to have this dependency in this repo, was it brought in transitively somewhere?